app/js/controllers/add-events-ctrl.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+angular
+    .module('app.controllers')
+    .controller('AddEventsCtrl', function($scope, Restangular) {
+      $scope.formFields = {};
+      $scope.submit = function() {
+
+        $http.post('https://api.tnyu.org/v3-test/events',{
```

Comment:
I think if you convert this to restangular like the one in`reimburse-ctrl` it will help you out by handling the URL and saving you from needing to set headers